### DISCUSSION TITLE
Show Gurobi solver output

### DISCUSSION THREAD

**Discussion by rtapia8484** [casadi/casadi#3632] [Q&A] [answered]
Hello,

I am using gurobi as the solver for a convex problem, I would like to see the solver output. Which for my problem means seeing some barrier interior point method details. I have tried using the 'verbose'=true , but that did not work. How can I see the output?

Much appreciated.

**Comment by rtapia8484** [comment, marked as answer]
use the following options
options = {
        'verbose': True,  # Make CasADi verbose (prints information about the solver)
        'print_time': True,  # Print time information
        'warm_start_init_point': 'yes',
        'gurobi': {
            'OutputFlag': 1,  # Gurobi-specific option to enable detailed logging
            'LPWarmStart': 2,  # gurobi warm start 0: none, 1: pre
            'LogFile': 'glob_SCP_ca_R1_1.txt',  # Save the Gurobi log to a file
            'Threads': 4,  # Example of using 4 threads (optional)
        }
    }

**Reply by polcrub** [reply]
Hello,

When interfacing with gurobi through qpsol the 'warm_start_init_point' doesn't seem to exists. Is there any way to warm start GUROBI, for example, using the 'x0' input when calling the qpsol solver??

Thanks

**Reply by rtapia8484** [reply]
@polcrub, you are correct, it does not exist. I am not aware of a way to specify a warm start to Gurobi via casadi. 

You may want to try opti.set_initial(), but if I recall correctly this has no effect when using Gurobi as a solver.
